csrc/aio/py_test/parse_aio_stats.py

```diff
@@ -107,12 +85,7 @@ def get_metric(file, metric):
     with open(file) as f:

```

Comment:
It might be useful to document this parsing method with the assumptions of the log format and some sample lines.

Reply:
Done.